### PR TITLE
Meta Station: Add a fully functional TEG room

### LINKED ISSUE #32923: Meta map TEG layout (or lackthereof) is painful
## Description

TEG on Meta is basically just stapled to the outside of the station, with no burn chamber, canisters or pipes ready.

This makes it incredibly trivial to steal TEG parts, sabotage it or have it be bombarded by meteors.

### PR DESCRIPTION
**Pull request by scrivoy**
## About the PR

This PR adds a dedicated TEG room to the "Meta" station, improving the layout and functionality of the area.
This PR is intended to close #32923.

## Why / Balance

Previously, the Meta station only provided an open space for the TEG, located in space without a dedicated room. Engineers were required to construct the setup entirely from scratch, which was both time-consuming and less accessible for newer players.
This PR introduces a pre-built TEG room with essential infrastructure, making it easier to set up and manage power generation. The inclusion of a dedicated space enhances usability and supports more streamlined gameplay for all skill levels.

## Technical details

Main Atmos Room:
- Move the gas chambers up one tile to make room for a two tile wide entrance area.
- Adjusted pipes and wiring

TEG Entrance Area:
- Piping for Distro, Waste and Burn-mix
- Own SMES and substation
- Firelocks
- External airlocks (as before)

TEG "Office":
- Power monitoring console
- Crate with metal for piping
- Some tools to get the job done
- Locker with Atmo fire suit, Atmo gas mask and O2/N2 tanks

TEG Room:
- Burn chamber with air alarm and two passive vents
- A freezer, some gas canisters and welding fuel
- Blast doors with linked two-way lever. To open one or two doors
- Outside area for cold loop and exhaust vent
- External airlock for quick access
- Meteor protection

## Media

Gas chambers adjusted to fit TEG entrance.
GasChambers_Adjusted

New TEG room with SMES, substation, and office.
TEGRoom_SMES_Office

Burn chamber and cold loop area with meteor protection.
BurnChamber_ColdLoop

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by scrivoy** [comment]
> Are you still working on this

Hi, yes I am still on this.
I just had a little much reallife going on the last days.

**Comment by scrivoy** [comment]
I used the TEG room from Marathon Station as my inspiration as it is my personal favourite room.
It has everything you need, plenty of room to play and not too big like the Bagel Station.
I had to move the eastern gas chambers up a bit so that the path to the TEG was not just one tile wide, which did not look good.
The outer walls are WIP and will get some windows.

**Comment by scrivoy** [comment]
this is the current status

**Comment by scrivoy** [comment]
I have to redesign it a bit, but it should not take too long.

**Comment by scrivoy** [comment]
this is the status as of today

**Comment by scrivoy** [comment]
this is the status as of today

**Comment by scrivoy** [comment]
I have to do some meteor and space debris protection and then its done, I think.

**Comment by scrivoy** [comment]
The pull request is complete, yay!

**Comment by scrivoy** [comment]
I have no idea what this means in the Build process:

[ERRO] map: Error in map file: found local entity UID '0' which does not exist.

**Comment by scrivoy** [comment]
I just had a look at the screenshots again and it could be that i forgot to put HV wires below the substation. I'll check that when kids are sleeping.

File: Resources/Maps/meta.yml [hunk 1]
```diff
@@ -11115,7 +11115,7 @@ entities:
       pos: -42.5,29.5
       parent: 5350
     - type: Door
-      secondsUntilStateChange: -25260.242
+      secondsUntilStateChange: -25306.475
```
Comment (scrivoy) [inline Resources/Maps/meta.yml:11118]: can someone explain what this is, please?

**Comment by Emisse** [comment]
map test fail real

**Comment by Emisse** [comment]
image
remove invalids from map file

**Comment by scrivoy** [comment]
> remove invalids from map file
ok i removed it, there were 4 entries. can you explain what that was? so i know for next time. thanks!

**Comment by Emisse** [comment]
if you delete 1/2 of a linked device it will serialize to an invalid state

**Comment by scrivoy** [comment]
ah okay, i understand. thanks!

And what about this build error? it is too difficult for me to understand at the moment.

**Comment by Emisse** [comment]
looks erroneous

**Pull request merged**